documentation/src/main/asciidoc/topics/proc_adding_jgroups_extras.adoc

```diff
@@ -0,0 +1,48 @@
+[id='adding-jgroups-extras_{context}']
+= Adding Additional JGroups Dependencies
+{brandname} includes default JGroups stacks that use discovery protocol implementations that are specific to cloud providers.
+These implementations require artifacts that {brandname} does not include by default.
+
+For this reason, if you plan to use these JGroups stacks, you need to add additional dependencies to your project.
+
+[%header,cols=3*]
+|===
+|File name
+|Discovery protocol
+|Additional dependency
+
```

Comment:
should we add a version column? Using the latest version may not work since all 3 projects are already using JGroups 5 and we are using JGroups 4

Reply:
the xml files are stored inside the infinispan-core.jar. I doubt users will extract those files from the jar to read the comments :)